web/Views/Landlord/Invite.cshtml

```diff
@@ -0,0 +1,19 @@
+@{
+    ViewData["Title"] = "Accept Invite";
+}
+
+@if (TempData["InviteLinkToAccept"] != null)
```

Comment:
Can you make a view model for this rather than using TempData?

TempData is great for stuff like flash messages that you want to display on another page, but they're not typed so it's better to use a view model for stuff that you'll always display.

Reply:
yes sure ok